src/docs/dev_tasks.md

```diff
@@ -265,3 +265,15 @@ Pants allows more fine grained cache management, although it then becomes the re
           if self.artifact_cache_writes_enabled():
             self.update_artifact_cache((vt, [output_location]))
 
+Recording Target Specific Data
+------------------------------
+If you would like to track target information like the targets being run, their
+run times, or some other target-specific piece of data, `run_tracker` has this
```

Comment:
s/has this ability by calling/provides this ability via/

In the original phrasing it sounds like the run_tracker calls the method, not that it's a method on the run_tracker.